Apps/Laboratory/src/Models/PublicCovidTestResponse.cs

```diff
@@ -15,84 +15,63 @@
 // -------------------------------------------------------------------------
 namespace HealthGateway.Laboratory.Models
 {
-    using System;
+    using System.Collections.Generic;
+    using System.Linq;
     using System.Text.Json.Serialization;
+    using HealthGateway.Common.Models.PHSA;
 
     /// <summary>
-    /// The representation of a COVID-19 test result for public access.
+    /// Represents the result from querying COVID-19 test responses in public.
     /// </summary>
     public class PublicCovidTestResponse
     {
         /// <summary>
-        /// Gets or sets the first name and last initial of the patient.
+        /// Initializes a new instance of the <see cref="PublicCovidTestResponse"/> class.
         /// </summary>
-        [JsonPropertyName("patientDisplayName")]
-        public string PatientDisplayName { get; set; } = string.Empty;
+        public PublicCovidTestResponse()
+        {
+        }
```

Comment:
what is the purpose to declare the default constructor?

Reply:
C# only creates a parameterless constructor if you don't have any other constructors.  I added this one so I could initialize the return value without passing in an empty list.

Reply:
I've updated this constructor so that it initializes the list instead of leaving it null, which was an oversight.